agi.go

```diff
@@ -408,6 +413,9 @@ func (a *AGI) SayDateTime(when time.Time, escapeDigits string, format string) (d
 
 // SayNumber plays the given number.
 func (a *AGI) SayNumber(number string, escapeDigits string) (digit string, err error) {
+	if len(escapeDigits) < 1 {
```

Comment:
This is fine, but it's a little weird to check a string value in this way.  If AGI really needs the empty quotes, more usual would be something like this.  I would add the comment, too, since it does not make immediate sense, in reading the code:

```go
// NOTE: AGI needs empty double quotes hold the place of the empty value in the line
if escapeDigits == "" {
  escapeDigits = `""`
}
```